src/libMessage/Messenger.cpp

```diff
@@ -850,6 +850,25 @@ bool SerializeToArray(const T& protoMessage, vector<unsigned char>& dst,
                                        protoMessage.ByteSize());
 }
 
+template <class T>
+bool RepeatableToArray(const T& repeatable, vector<unsigned char>& dst,
+                       const unsigned int offset) {
+  int tempOffset = offset;
+  for (int i = 0; i < repeatable.size(); ++i) {
```

Comment:
Will it work if we change to range-based for loop? `for (const auto& element : repeatable)`

Reply:
Ok, i will update.